mappings/net/minecraft/entity/mob/EntityHusk.mapping

```diff
@@ -1,7 +1,11 @@
 CLASS none/zo net/minecraft/entity/mob/EntityHusk
+	METHOD B (Lnone/sg;)Z
+		ARG 0 entity
 	METHOD G getSoundAmbient ()Lnone/nk;
 	METHOD J getLootTableId ()Lnone/kp;
 	METHOD b registerTagUpgraders (Lnone/pf;)V
 		ARG 0 handler
 	METHOD bV getSoundHurt ()Lnone/nk;
 	METHOD bW getSoundDeath ()Lnone/nk;
+	METHOD df getSoundStep ()Lnone/nk;
+	METHOD dg getSkull ()Lnone/aes;
```

Comment:
It might very well be the drop! Doesn't have to be a list... I think getChargedCreeperDrop, or getDroppedSkull, might work better.